src/parpecommon/hdf5Misc.cpp

```diff
@@ -797,4 +804,32 @@ bool hdf5DatasetExists(const H5::H5File &file, const std::string &datasetName)
     return hdf5DatasetExists(file.getId(), datasetName.c_str());
 }
 
+std::vector<std::string> hdf5Read1dStringDataset(
+        H5::H5File const& file, const std::string &datasetPath)
+{
+    auto lock = hdf5MutexGetLock();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'lock' is assigned a value that is never used.](https://app.codacy.com/manual/dweindl/parPE/pullRequest?prid=4922781)